src/Microsoft.ML.Data/Transforms/SkipTakeFilter.cs

```diff
@@ -270,4 +270,30 @@ protected override bool MoveManyCore(long count)
             }
         }
     }
+
+    public static class SkipFilter
+    {
+        /// <summary>
+        /// A helper method to create <see cref="SkipFilter"/> for public facing API.
```

Comment:
>A helper method to create <see cref="SkipFilter"/> for public facing API. [](start = 12, length = 73)

You cannot create a `SkipFilter` since there is no such instantiable type. I might rather describe what it does.